non-critical-infra/modules/hydra-queue-runner-v2.nix

```
@@ -0,0 +1,145 @@
+{
+  config,
+  pkgs,
+  lib,
+  unstable,
+  ...
+}:
+let
+  cfg = config.services.hydra-queue-runner-v2;
+
+  format = pkgs.formats.toml { };
+  configFile = format.generate "hydra-queue-runner-v2.toml" cfg.settings;
+in
+{
+  options = {
+    services.hydra-queue-runner-v2 = {
+      enable = lib.mkEnableOption (lib.mdDoc "QueueRunner");
```

```suggestion
      enable = lib.mkEnableOption "QueueRunner";
```

I don't think we need `lib.mdDoc` any more. All option docs should be markdown anyway.